website/docs/d/mobile_network_site.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+subcategory: "Mobile Network"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_mobile_network_site"
+description: |-
+  Get information about a Mobile Network Site.
+---
+
+# azurerm_mobile_network_site
+
+Get information about a Mobile Network Site.
+
+## Example Usage
+
+```hcl
+data "azurerm_mobile_network" "example" {
+  name                = "example-mn"
+  resource_group_name = "example-rg"
+}
+
+data "azurerm_mobile_network_site" "example" {
+  name              = "example-mns"
+  mobile_network_id = data.azurerm_mobile_network.example.id
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Mobile Network Site. Changing this forces a new Mobile Network Site to be created.
+
+* `mobile_network_id` - (Required) the ID of the Mobile Network which the Mobile Network Site belongs to. Changing this forces a new Mobile Network Site to be created.
```

Comment:
We can remove the force new warnings here since this is the datasource.

```suggestion
* `name` - (Required) The name which should be used for this Mobile Network Site.

* `mobile_network_id` - (Required) the ID of the Mobile Network which the Mobile Network Site belongs to.
```